test/test_scanner.py

```diff
@@ -994,6 +1008,18 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "gstreamer",
                         "0.10",
                     ),
+                    (
+                        "https://kojipkgs.fedoraproject.org/packages/wireshark/1.10.13/1.fc20/x86_64/",
```

Comment:
Alphabetize.   (As best you can if some tests are in the incorrect position.)

Reply:
I have tried my best, but I will submit a PR for arranging the tests in alphabetical order.